audapter_viewer/set_subjOstParams_auto.m

```diff
@@ -61,7 +57,11 @@
 
 % file name
 if isfield(expt, 'trackingFileName')
-    fileName = expt.trackingFileName;
+    if iscell(expt.trackingFileName) && length(expt.trackingFileName) > 1
+        fileName = expt.trackingFileName{1}; 
+    else
+        fileName = expt.trackingFileName;
+    end
```

Comment:
actually now that I look at this I think this is wrong, but it worked with taimComp anyway. I will fix it